rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml

```
@@ -2,7 +2,7 @@ title: DLL Sideloading via DeviceEnroller.exe
 id: e173ad47-4388-4012-ae62-bd13f71c18a8
 status: experimental
 description: |
-    Detects the use of the PhoneDeepLink parameter to potentially sideload a DLL file that doesnt exist. This non-existent DLL file is named "ShellChromeAPI.dll". 
+    Detects the use of the PhoneDeepLink parameter to potentially sideload a DLL file that does not exist. This non-existent DLL file is named "ShellChromeAPI.dll". 
```

I reverted this change because it's fixed in #4007 and the file got renamed. So to avoid conflict :)

```suggestion
    Detects the use of the PhoneDeepLink parameter to potentially sideload a DLL file that doesnt exist. This non-existent DLL file is named "ShellChromeAPI.dll". 
```